Rakefile

```diff
@@ -0,0 +1,39 @@
+#
+# constants
+#
+CHEF_VERSION = '11.16.4'
```

Comment:
I added and deleted this line a number of times. I decided to pin it to Chef 11 for the moment, as you said, it is a future improvement. I have not updated the README file, which the note will need to be made there about this improvement.